types/auth0/auth0-tests.ts

```diff
@@ -191,26 +191,60 @@ auth.tokens
     });
 
 // Password Grant
-auth.passwordGrant({username: 'username', password: 'password'}).then((response: auth0.TokenResponse) => { console.log(response); });
-auth.passwordGrant({username: 'username', password: 'password'}, (err, response: auth0.TokenResponse) => { console.log(response); });
-auth.passwordGrant({username: 'username', password: 'password'}, { forwardedFor: '12.34.56.78' }).then((response: auth0.TokenResponse) => { console.log(response); });
-auth.passwordGrant({username: 'username', password: 'password'}, { forwardedFor: '12.34.56.78' }, (err, response: auth0.TokenResponse) => { console.log(response); });
+auth.passwordGrant({ username: 'username', password: 'password' }).then((response: auth0.TokenResponse) => {
+    console.log(response);
+});
+auth.passwordGrant({ username: 'username', password: 'password' }, (err, response: auth0.TokenResponse) => {
+    console.log(response);
+});
+auth.passwordGrant({ username: 'username', password: 'password' }, { forwardedFor: '12.34.56.78' }).then(
+    (response: auth0.TokenResponse) => {
+        console.log(response);
+    },
+);
+auth.passwordGrant(
+    { username: 'username', password: 'password' },
+    { forwardedFor: '12.34.56.78' },
+    (err, response: auth0.TokenResponse) => {
+        console.log(response);
+    },
+);
```

Comment:
```suggestion
auth.passwordGrant({ username: 'username', password: 'password' }).then(response => {
    response; // $ExpectType TokenResponse
});
auth.passwordGrant({ username: 'username', password: 'password' }, (err, response) => {
    response; // $ExpectType TokenResponse
});
auth.passwordGrant({ username: 'username', password: 'password' }, { forwardedFor: '12.34.56.78' }).then(response => {
    response; // $ExpectType TokenResponse
});
auth.passwordGrant({ username: 'username', password: 'password' }, { forwardedFor: '12.34.56.78' }, (err, response) => {
    response; // $ExpectType TokenResponse
});
```
please retain format mandated by DT setup (unless there is a specific reason to pick other format, like autogenerated content). With those assertion we will test more real life usage (one should not use specific type, unless being forced to)